robottelo/ui/contentviews.py

```diff
@@ -540,12 +540,9 @@ def fetch_puppet_module(self, cv_name, module_name):
     def copy_view(self, name, new_name):
         """Copies an existing Content View."""
         self.search_and_click(name)
-        self.edit_entity(
-            locators['contentviews.copy'],
-            locators['contentviews.copy_name'],
-            new_name,
-            locators['ak.copy_create']
-        )
+        self.perform_entity_action('Copy')
+        self.assign_value(locators['ak.copy_name'], new_name)
```

Comment:
@oshtaier, yes you are right the copy functionality can be moved to base class as delete, but we have many tests failing on this.
We have to open an issue to standardize the copy, as we have to wait until, the things became more stable, what do you think?